init.lua

```diff
@@ -2,16 +2,73 @@ channels = {}
 channels.huds = {}
 channels.players = {}
 
+-- workaround for settings:get*() defaults not working
+local function notnil_or(d,v)
```

Comment:
There are no default params in 0.4.x, thus it's "not working" for you.

I'd prefer to remove this function and just inline it:
```Lua
channels.allow_global_channel = minetest.settings:get_bool("channels.allow_global_channel") ~= false
channels.disable_private_messages = minetest.settings:get_bool("channels.disable_private_messages") == true
```